App/navigation/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {NavigationContainer} from '@react-navigation/native';
+import {Routes} from '../constants';
+import {
+  HomeScreen,
+  VideoScreen,
+  RadioScreen,
+  ScheduleScreen,
+  SearchCenterScreen,
+} from '../screens';
+import {OrgHeader} from '../components';
+import {Header} from '../components';
+import FontTelloIcon from '../components/FontTelloIcon';
+import {ApolloProvider} from '@apollo/react-hooks';
+import client from './apolloClient';
+import {Colors} from '../constants';
+const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
+
+const MyTab = () => {
+  return (
+    <Tab.Navigator
+      headerMode="screen"
+      screenOptions={({route}) => ({
+        tabBarIcon: ({focused, color, size}) => {
+          let iconName;
+          //TODO: remove from here
+          if (route.name === 'Home') {
+            iconName = focused ? 'home' : 'home';
+          } else if (route.name === 'Video') {
+            iconName = focused ? 'videocam-1' : 'videocam-1';
+          } else if (route.name === 'Radio') {
+            iconName = focused ? 'signal' : 'signal';
+          } else if (route.name === 'Schedule') {
+            iconName = focused ? 'calendar-empty' : 'calendar-empty';
+          }
+
```

Comment:
move these icons to tab screen tags

Reply:
I need to discuss these again for better understanding 
Secondly, I have replaced these strings with enums